Wikimate.php

```diff
@@ -127,6 +154,77 @@ protected function initRequests()
 		$this->session->useragent = $this->useragent;
 	}
 
+	/**
+	 * Sends a GET or POST request in JSON format to the API.
+	 *
+	 * This method handles maxlag errors as advised at:
+	 * {@see https://www.mediawiki.org/wiki/Special:MyLanguage/Manual:Maxlag_parameter)
+	 * The request is sent with the current maxlag value (default 5 seconds).
```

Comment:
Would it be worth mentioning `MAXLAG_DEFAULT` here, just to make sure any future changes to the value don't miss this comment? For example:

```suggestion
	 * The request is sent with the current maxlag value (default: 5 seconds, per MAXLAG_DEFAULT).
```